### DISCUSSION TITLE
Using watchtower as a container (re)start scheduler

### DISCUSSION THREAD

**Discussion by IT-VBFK** [containrrr/watchtower#1852] [Q&A]
Hi

I have a scenario where I have a node.js app, but no web api. This means I use the container with the `--rm` flag to perform an action and clean up afterwards.

I was wondering if I could use watchtower as a docker container "spawner" with schedule. E.g. "Watchtower, please start the container 'my-action:latest', with schedule "0 0 7 ? * WED" and remove it afterwards".

Which will result in (docker speaking): `docker run --rm -d my-action:latest`, every wednesday at 07:00.

**Comment by piksel** [comment]
I don't think that would work, since watchtower only operates on existing containers and only recreates them when their images are updated. 

Why not simply use Cron instead?
